elfloader-tool/src/arch-riscv/boot.c

```diff
@@ -59,7 +59,7 @@ unsigned long l2pt[PTES_PER_PT] __attribute__((aligned(4096)));
 unsigned long l2pt_elf[PTES_PER_PT] __attribute__((aligned(4096)));
 #endif
 
-char elfloader_stack_alloc[BIT(CONFIG_KERNEL_STACK_BITS)];
+char elfloader_stack_alloc[BIT(CONFIG_KERNEL_STACK_BITS)] __attribute__((section(".data"))) ;
```

Comment:
Can you add a comment here that explains why this is in the data section? Could be the same as in the commit comment.